content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -107,6 +114,21 @@ have changed, the container will be restarted in order to resize its memory.
 
 <!-- steps -->
 
+## Limitations
```

Comment:
This belongs further up. Make this either a subheading of the prerequisites heading or a main heading before the prerequisites heading.